server/models/groups/updateGroupName.js

```diff
@@ -0,0 +1,43 @@
+const { pool } = require('../database')
+
+/**
+ *z
+ * @param {*} param0
+ * @param {Object} param0.params
+ * @param {Object} param0.body
+ * @param {Object} param0.username
+ * @return {Promise}
+ *
+ */
+
+function updateGroupName({ params, body, username }) {
+  return new Promise((resolve, reject) => {
+    const { group_id: groupId } = params
+    const { new_group_name: newGroupName } = body
+    pool.query(
+      `SELECT is_group_moderator FROM user_groups WHERE group_id=? AND username=?`,
+      [groupId, username],
```

Comment:
Due to less entries for a username, as compared to that of group_id, this would be more optimal I guess
```suggestion
      `SELECT is_group_moderator FROM user_groups WHERE username=? and group_id=? `,
      [username,groupId],
```